extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -39,10 +38,18 @@ import {
   escapeCssSelectorIdent,
 } from '../../../src/dom';
 import {clamp} from '../../../src/utils/math';
+import {
+  concat as concatTransition,
+  numeric,
+  scale,
+  setStyles as setStylesTransition,
+  translate,
+} from '../../../src/transition';
```

Comment:
I'm going to give a vote for using the default import in this case. When both `transition` and `styles` functions are imported, I think I find it easier to keep track of these methods when they're namespaced like `tr.setStyles(blah)` and `st.setStyles(blah)` but do what you think is most ergonomic.